views/email_preview_template.ejs

```diff
@@ -0,0 +1,225 @@
+<% include _header.ejs %>
+
+<div class="container">
+    <div class="col-lg-2"></div>
+
+    <div class="col-lg-8 email-preview-container">
+        
+        <!-- EMAIL PREVIEW TEMPLATE STARTS  -->
+
+        <div class="col-lg-12">
+            <h3 class="email-preview-title">EMail Preview Mode</h3>
+        </div>
+
+        <div class="col-lg-12 email-preview-fields-container">
+
+            <div class="form-group">
+                <label class="control-label col-lg-2" for="email-to">To</label>
+
+                <div class="col-lg-10 email-to">
+                  <label class="control-label field-input" for="email-to"> <%= allUserIds[participantCount].user.get("emails")[0].value %> </label>
```

Comment:
If we're only ever using ``allUserIds[participantCount]`` rather than any of the other users in ``allUserIds``, then let's just pass the single user data set to the template context.